kubernetes/k8s_spec.yml

```diff
@@ -20,21 +20,27 @@ spec:
         command: ["/usr/sbin/sshd"]
         args: ["-D"]
         tty: true
-        image: leogao2/gpt-neox
+        image: leogao2/gpt-neox:synced-deployment
```

Comment:
This should be changed to `leogao2/gpt-neox:main` once merged into `main` branch